grpc/src/main/java/com/linecorp/armeria/server/grpc/ArmeriaServerCall.java

```diff
@@ -597,7 +593,11 @@ private void closeListener(Status newStatus, boolean completed, boolean setRespo
 
             if (!clientStreamClosed) {
                 clientStreamClosed = true;
-                deframedRequest.abort();
+                if (newStatus.isOk()) {
+                    req.abort();
```

Comment:
Because `defamedRequest` aborts upstream (`req`) using `upstream.cancel()` that can not propagate a cause to upstream.
So I used `req.abort(cause)` to propagate the cause to downstream and `RequestLog`.

Reply:
Ah I misread the code 😅  👍